### PR TITLE
:sparkles: Add genre, type, author machines

### PR DESCRIPTION
**Pull request by pierman1**

### CODE REVIEW

File: app/components/parts/author/index.js [hunk 1]
```diff
@@ -0,0 +1,205 @@
+import html from 'choo/html';
+import styles from '../parts.css';
+
+console.log(styles);
```
Comment (rijkvanzanten) [inline app/components/parts/author/index.js]: Trailing `console.log`

File: app/components/parts/author/index.js [hunk 2]
```diff
@@ -0,0 +1,205 @@
...
+        id,
+        value: 'Test'
+      });
+    }
+  },
+  color: '#da008c'
+};
+
+export default () => html`
+<svg class=${styles.svg} xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="295" height="225" viewBox="0 0 295 225">
```
Comment (rijkvanzanten) [inline app/components/parts/author/index.js]: Expected indentation of 2 spaces. Found 0

File: app/components/parts/genre/index.js [hunk 3]
```diff
@@ -14,74 +15,79 @@ export const modalSettings = {
...
-          </g>
+<svg class=${styles.svg} xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="218" height="225" viewBox="0 0 218 225">
```
Comment (rijkvanzanten) [inline app/components/parts/genre/index.js]: indentation

File: app/routes/body.css [hunk 4]
```diff
@@ -7,3 +7,14 @@
...
   margin: 0;
 }
+
+.machine_ul {
+  font-size: 0;
+  margin: 0;
+  padding: 0;
+}
+
+li {
```
Comment (rijkvanzanten) [inline app/routes/body.css]: ❌

File: app/routes/body.css [hunk 5]
```diff
@@ -7,3 +7,14 @@
 h1 {
   margin: 0;
 }
+
+.machine_ul {
+  font-size: 0;
```
Comment (rijkvanzanten) [inline app/routes/body.css]: display: flex; flex-direction: column;

**Review by Mimaaa — APPROVED** [review]

**Pull request merged**